components/dashboard/src/components/DropDown2.tsx

```diff
@@ -207,3 +240,40 @@ export const DropDown2: FunctionComponent<DropDown2Props> = (props) => {
         </div>
     );
 };
+
+type DropDown2SelectedElementProps = {
+    iconSrc: string;
+    loading?: boolean;
+    title: ReactNode;
+    subtitle: ReactNode;
+    htmlTitle?: string;
+};
+
+export const DropDown2SelectedElement: FC<DropDown2SelectedElementProps> = ({
+    iconSrc,
+    loading = false,
+    title,
+    subtitle,
+    htmlTitle,
+}) => {
+    return (
+        <div className={classNames("flex items-center", loading && "animate-pulse")} title={htmlTitle}>
+            <div className="mx-2 my-3 flex-shrink-0">
+                <img className="w-8 filter-grayscale" src={iconSrc} alt="logo" />
+            </div>
+            <div className="flex-col ml-1 flex-grow">
+                {loading ? (
+                    <div className="flex-col space-y-2">
+                        <div className="bg-gray-300 dark:bg-gray-500 h-4 w-24" />
+                        <div className="bg-gray-300 dark:bg-gray-500 h-2 w-40" />
```

Comment:
**praise**: Nice touch on mimicking the actual layout with different height and width. 👏